### DISCUSSION TITLE
Tutorial to install SQLPage on Synology NAS with MySQL Database

### DISCUSSION THREAD

**Discussion by ThierryM** [sqlpage/SQLPage#290] [Show and tell]
Hi,
I have realized, in french, an "How to" use SQLPage with a Synology NAS an MySQL databases in this article : https://lofurol.fr/joomla/logiciels-libres/104-bases-de-donnees/349-sqlpage-utilisation-sur-un-nas-synology-avec-docker-et-mysql
I hope it will help new users to use SQLPage that I discovered recently.
Regards,

Thierry

**Comment by lovasoa** [comment]
Merci, c'est très utile !

**Reply by ThierryM** [reply]
Merci à toi pour SQLPage ;-) !
Par contre, tu as vu l'erreur sur AUTOINCREMENT dans le fichier 0001_create_users_table.sql ? Est-ce une particularité de MySQL ?

**Reply by lovasoa** [reply]
Oui, le tutoriel officiel explique comment lancer un projet avec SQLite, la base de données qui est intégrée directement dans SQLPage. 

D'autres bases de données ont des syntaxes (légèrement) différentes.
